tutorials/international/index.md

```diff
@@ -100,6 +150,99 @@ static MaterialLocalizations of(BuildContext context) {
 tooltip: MaterialLocalizations.of(context).backButtonTooltip,
 ```
 
+### Using the bundled LocalizationsDelegates
+
+To keep things as small and uncomplicated as possible, the flutter
+package includes implementations of the `MaterialLocalizations` and
+`WidgetsLocalizations` interfaces that only provide US English
+values. These implementation classes are called
+`DefaultMaterialLocalizations` and
+`DefaultWidgetsLocalizations`. They're included automatically unless a
+different delegate of the same base type is specified with the app's
+`localizationsDelegates` parameter.
+
+The flutter_localizations package includes multi-language
+    implementations of the localizations interfaces called `GlobalMaterialLocalizations`
+and `GlobalWidgetsLocalizations`. International apps must specify
+localization delegates for these classes as described in the "Setting
+up an Internationalized app" section above.
+
+```dart
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+new MaterialApp(
+ localizationsDelegates: [
+   // ... app-specific localization delegate[s] here
+   GlobalMaterialLocalizations.delegate,
+   GlobalWidgetsLocalizations.delegate,
+ ],
+ supportedLocales: [
+    const Locale('en', 'US'), // English
+    const Locale('he', 'IL'), // Hebrew
+    // ... other locales the app supports
+  ],
+  // ...
+)
+```
+
+The global localization delegates construct locale-specific instances
+of the corresponding classes. For example
+`GlobalMaterialLocalizations.delegate` is a `LocalizationsDelegate`
+that produces an instance of `GlobalMaterialLocalizations`.
+
+Currently the global localization classes support about 15 langauges
+(more are coming). Applications can add support for additional
```

Comment:
be specific about num of languages, add "as of YY date", and remove promise that more are coming. Even better, link to the directory that contains all the languages.